### PR TITLE
feat: disables form submit button, if form does not have any changes (DEV-3829)

### PR DESCRIPTION
**Pull request by irmastnt**
resolves DEV-3829

### CODE REVIEW

File: apps/dsp-app/src/app/project/reusable-project-form/reusable-project-form.component.ts [hunk 1]
```diff
@@ -65,13 +68,18 @@ export class ReusableProjectFormComponent implements OnInit {
   readonly keywordsValidators = [Validators.minLength(3), Validators.maxLength(64)];
   readonly descriptionValidators = [Validators.minLength(3), Validators.maxLength(40960)];
 
+  get hasChanges(): boolean {
```
Comment (derschnee68) [inline apps/dsp-app/src/app/project/reusable-project-form/reusable-project-form.component.ts:71]: please don't use this: Angular form knows when the form has changed value.
Something like form.dirty

**Review by derschnee68 — CHANGES_REQUESTED** [review]
in everything you have done, I think you are not leveraging the angular form.

<button [disabled]="form.invalid" /> is enough, no ?

File: apps/dsp-app/src/app/project/reusable-project-form/reusable-project-form.component.ts (hunk 1 above)
Comment (irmastnt) [reply apps/dsp-app/src/app/project/reusable-project-form/reusable-project-form.component.ts:71]: @derschnee68 checking if the form is dirty is not enough, it only check direct fields of the form but not the composite components

**Comment by irmastnt** [comment]
Requirements were changed. Closing this PR.

**Pull request closed without merging**